seed/seed.ts

```diff
@@ -0,0 +1,49 @@
+import { dbConnection } from "../mongoConnection";
+import { addPlace, getAllPlaces } from "../data/places";
+
+async function main() {
+	const db = await dbConnection();
+
+	// wipes the db so that we can test that things actually work later on in this script
+	console.log("Wiping the database clean...");
+	await db.dropDatabase();
```

Comment:
should a test script really work with our development db if it will wipe it clean at every run? maybe we should create another database just for testing this seed script.

Reply:
also
```
Wiping the database clean...
/home/david/Documents/dev/peer-server/node_modules/mongodb/src/cmap/connection.ts:740
          callback(new MongoServerError(document));
                   ^
MongoServerError: user is not allowed to do action [dropDatabase] on [development.]
```

Reply:
@carpenterd777 For some reason, you weren't an admin in the Database access portion, so I went ahead and fixed that in atlas. Can you please try again and lmk if it works?

Reply:
As for the script, we can definitely have a dedicated db for seeding, but I assumed that development would be the environment for that since seeding would only be necessary when testing some kind of functionality within the application during the development stage. Let me know what you think.

Reply:
why would we need to seed the database more than one time? This is a remote database that anyone can access

Reply:
So, if you're writing new db functions and testing them, sometimes you'll fluff up the data that's in your test db. So a seed is kinda like a fresh start. This was just a little script to get us started. I didn't think this would remain as is during the complete development of this project, and I anticipate this will change since we'll probably create our own db's for testing and things like that. But I didn't want to overcomplicate it from the get-go. I should have explained that better in my note up top, sorry about that.